pwnlib/fmtstr.py

```diff
@@ -36,15 +36,15 @@
     >>> def exec_fmt(payload):
     ...     p = process(program)
     ...     p.sendline(payload)
-    ...     return p.recvall()
+    ...     return p.recvall(timeout=0.1)
     ...
     >>> autofmt = FmtStr(exec_fmt)
     >>> offset = autofmt.offset
     >>> p = process(program, stderr=PIPE)
     >>> addr = unpack(p.recv(4))
     >>> payload = fmtstr_payload(offset, {addr: 0x1337babe})
     >>> p.sendline(payload)
-    >>> print hex(unpack(p.recv(4)))
+    >>> print hex(unpack(p.recv(4, timeout=0.1)))
```

Comment:
Why the timeout here (and above)?

Reply:
oops, this was for testing. I had some cases where I made the program crash (?) and this hung forever/a long time, can probably remove it now